janusgraph-cassandra/src/test/java/org/janusgraph/diskstorage/cassandra/thrift/CassandraThriftStoreManagerTest.java

```diff
@@ -0,0 +1,56 @@
+// Copyright 2017 JanusGraph Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.janusgraph.diskstorage.cassandra.thrift;
+
+import static org.junit.Assert.fail;
+
+import org.janusgraph.CassandraStorageSetup;
+import org.janusgraph.diskstorage.configuration.ModifiableConfiguration;
+import org.junit.Test;
+
+public class CassandraThriftStoreManagerTest {
+
+    @Test
+    public void shouldValidateConfigOptionFrameSizeMb() {
+
+        try {
```

Comment:
Please separate each block into a separate test method, and either use the `@Test(expected = MyRuntimeException.class)` annotation or the `ExpectedException` rule.

For more info, please see:

* https://blog.goyello.com/2015/10/01/different-ways-of-testing-exceptions-in-java-and-junit/
* http://junit.org/junit4/javadoc/4.12/org/junit/rules/ExpectedException.html

The resulting code will be much shorter and easier to read.

Reply:
Done.